db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/cleanup.rb

```diff
@@ -1,26 +1,41 @@
 INVENTORY_FILE = 'inventory.yaml'.freeze
 RHEL_SUBSCRIBE_INVENTORY = 'rhel_subscribe_inventory.yaml'.freeze
 
+def find_vm_by_tag(tag)
+  tag = "/managed/miq_openshift_deploy/#{tag}_#{$evm.root['automation_task'][:id]}"
+  $evm.vmdb(:vm).find_tagged_with(:any => tag, :ns => "*")
```

Comment:
Is :ns => "*" needed given full /managed/ path?
(too lazy to check now, might well be, it's a messy API)